cleanup/macparakeet_cleanup/cli.py

```
@@ -0,0 +1,230 @@
+            prompt=system_override,
+        )
+
+    if args.mode == "rules":
+        out = clean_rules(text)
+        chosen = "rules"
+    elif args.mode == "llm":
+        _ensure()
+        try:
+            out = run_llm()
+            chosen = "llm"
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)